org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/ClasspathUpdateHandlerTest.java

```diff
@@ -73,7 +74,7 @@ public void testClasspathUpdateForMaven() throws Exception {
 		waitForBackgroundJobs();
 
 		ArgumentCaptor<ActionableNotification> argument = ArgumentCaptor.forClass(ActionableNotification.class);
-		verify(connection, times(1)).sendActionableNotification(argument.capture());
+		verify(connection, atLeastOnce()).sendActionableNotification(argument.capture());
```

Comment:
I'd like to know why exactly this is happening. Is it expected we get multiple calls?

Reply:
checking...

Reply:
I tried to run `org.eclipse.jdt.ls.core.internal.handlers.ClasspathUpdateHandlerTest.testClasspathUpdateForMaven` locally(on Mac and Windows) and the invocation times is always one. wired... 🤔

Reply:
You can try the following:
```
$ rm -rf ~/.m2/repository/org/apache/commons/commons-lang3
$ mvn clean verify
```